src/web/mjs/connectors/rawmangatop.mjs

```diff
@@ -0,0 +1,70 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class RawMangatop extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'rawmangatop';
+        super.label = 'Raw Manga (生漫画)';
+        this.tags = [ 'manga', 'raw', 'japanese' ];
+        this.url = 'https://rawmanga.top';
+
+        this.config = {
+            throttle: {
+                label: 'Throttle Requests [ms]',
+                description: 'Enter the timespan in [ms] to delay consecuitive HTTP requests.\nThe website may reject to many consecuitive requests.\nSlightly increase the value when getting 429 errors during manga list update.',
```

Comment:
Adjust the description, as i see it, the throttle value is not used for the manga list, so `\nSlightly increase the value when getting 429 errors during manga list update.` makes no sense.